LayoutTests/http/tests/inspector/dom/site-isolated-node.html

```diff
@@ -0,0 +1,43 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<!doctype html>
+<html>
+<head>
+<script type="text/javascript" src="../resources/inspector-test.js"></script>
+<script>
+function test()
+{
+    let suite = InspectorTest.createAsyncSuite("CommandLineAPI.$0.cross-frame");
+
+    let mainFrame = WI.networkManager.mainFrame;
+    let childFrames = Array.from(mainFrame.childFrameCollection);
+    InspectorTest.expectEqual(childFrames.length, 1, "Page should have a subframe.");
+
+    let iframeNodeId;
+
+    suite.addTestCase({
+        name: "AttemptCrossFrame$0AccessFromMainFrame",
+        description: "Should not be able to access $0 node in different domain subframe from the main frame.",
+        test(resolve, reject) {
+            InspectorTest.log("Setting $0 to node within subframe.");
+            DOMAgent.setInspectedNode(iframeNodeId);
```

Comment:
wait so this is setting `$0` to be equal to the `<iframe>` node itself.  is that what we're trying to test?  or should we be querying within the `<iframe>` for the `<div id="rootDiv">` and then trying to set/get that to `$0`?